### PR TITLE
Add Containers and Leveled Lists Fixes load order

### PR DESCRIPTION
**Pull request by sean-kang**
WACCF, ACE, WAF, Lore worthy bandits should not overwrite leveled list changes of CLLF. It is the same rule as MLU load order.

### CODE REVIEW

**Comment by MacSplody** [comment]
> It is the same rule as MLU load order.

It could be put into the economy group.
And It can still be used with MLU, according to the author, as long as it's after MLU.
If it was added to the economy group an after would have to be added to `Trade & Barter.esp`.
To place it after `Containers and Leveled Lists Fixes.esp`, otherwise there would be sorting errors from patches.

**Comment by sean-kang** [comment]
I made changes as you suggested. Please check if it looks all right.

File: masterlist.yaml [hunk 1]
```diff
@@ -6910,6 +6910,19 @@ plugins:
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/18571/' ]
     after: [ 'Requiem.esp' ]
 
+  - name: 'Containers and Leveled Lists Fixes.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/26575/' ]
+    group: *economyGroup
+    after:
+      - 'LoreWorthyBandits.esp'
+      - 'Weapon AF.esp'
```
Comment (MacSplody) [inline masterlist.yaml]: Looks fine, it needs an after for Morrowloot Ultimate as well.

```suggestion
      - 'LoreWorthyBandits.esp'
      - 'MLU.esp'
      - 'Weapon AF.esp'
```

**Comment by sean-kang** [comment]
Updated. Thanks.

**Review by MacSplody — APPROVED** [review]

**Comment by MacSplody** [comment]
@sean-kang Thanks.

**Pull request merged**